--- node_modules/verda/lib/session/index.js	2021-03-11 17:52:56.000000000 +0300
+++ node_modules/verda/lib/session/index2.js	2021-03-11 17:52:13.000000000 +0300
@@ -39,11 +39,11 @@
     bindConfig(options) {
         this.config.bind(options);
         this.director.setCapacity(this.config.jCmd);
-        if (process.stderr.isTTY && process.stdout.isTTY) {
+        //if (process.stderr.isTTY && process.stdout.isTTY) {
             this.reporter = new console_1.default(this.config.verbosity);
             this.director.reporter = this.reporter;
             this.config.reporter = this.reporter;
-        }
+        //}
     }
     createSelfTrackingRule(dependency) {
         if (!this.config || !this.userSelfTrackingSet)
